spec/models/spree/variant_spec.rb

```diff
@@ -13,23 +13,19 @@
     let(:variant) { create(:variant) }
 
     it "invalidates the cache on update" do
-      travel_to 5.minutes.from_now do
-        current_time = Time.current
-        expect { variant.reload.update!(price: 30) }.to change { ingredient.reload.updated_at }.to(current_time)
-        expect(element.reload.updated_at).to eq(current_time)
-        expect(page_version.reload.updated_at).to eq(current_time)
-        expect(page.reload.updated_at).to eq(current_time)
-      end
+      expect {
+        variant.update!(price: 30)
```

Comment:
why does this trigger even though we only registered an `after_touch` callback...???

Reply:
Because we update the price and this runs callbacks that touch the variant. Updated to test updating a simple attribute instead.